### PR TITLE
Messaging tweaks for `InstallUncaughtExceptionHandler`

### PR DESCRIPTION
**Pull request by jglick**
Amending wording from @jtnord in #2095 etc. In particular, _in a bad way_ does not sound right to me.

### Testing done

N/A

### Proposed changelog entries

- N/A

### Maintainer checklist

Before the changes are marked as `ready-for-merge`:

- [ ] There are at least two (2) approvals for the pull request and no outstanding requests for change.
- [x] Conversations in the pull request are over, or it is explicit that a reviewer is not blocking the change.
- [x] Changelog entries in the pull request title and/or **Proposed changelog entries** are accurate, human-readable, and in the imperative mood.
- [x] Proper changelog labels are set so that the changelog can be generated automatically.
- [x] If the change needs additional upgrade steps from users, the `upgrade-guide-needed` label is set and there is a **Proposed upgrade guidelines** section in the pull request title (see [example](https://github.com/jenkinsci/jenkins/pull/4387)).
- [x] If it would make sense to backport the change to LTS, a Jira issue must exist, be a _Bug_ or _Improvement_, and be labeled as `lts-candidate` to be considered (see [query](https://issues.jenkins.io/issues/?filter=12146)).

<a href="https://gitpod.io/#https://github.com/jenkinsci/jenkins/pull/7622"></a>

### CODE REVIEW

File: core/src/main/java/hudson/init/impl/InstallUncaughtExceptionHandler.java
Comment (jglick) [inline core/src/main/java/hudson/init/impl/InstallUncaughtExceptionHandler.java:40]: https://www.cultofpedagogy.com/two-spaces-after-period/ LOL

**Review by MarkEWaite — APPROVED** [review]

**Review by NotMyFault — APPROVED** [review]
/label ready-for-merge

---

This PR is now ready for merge. We will merge it after ~24 hours if there is no negative feedback.
Please see the [merge process documentation](https://github.com/jenkinsci/jenkins/blob/master/docs/MAINTAINERS.adoc#merge-process) for more information about the merge process.
Thanks!

**Pull request merged**
